tests/sentry/middleware/test_access_log_middleware.py

```diff
@@ -109,3 +125,33 @@ class TestAccessLogFail(LogCaptureAPITestCase):
     def test_access_log_fail(self):
         self.get_error_response(status_code=500)
         self.assert_access_log_recorded()
+
+
+class TestOrganizationIdPresent(LogCaptureAPITestCase):
```

Comment:
If you want to use the `get_success_response()` to remove boilerplate, you can do:
```py
class TestOrganizationIdPresent(LogCaptureAPITestCase):
    endpoint = "sentry-api-0-organization-stats-v2"

    def setUp(self):
        self.login_as(user=self.user)

    def test_org_id_populated(self):
        self.get_success_response(
            self.organization.slug,
            qs_params={
                "project": [-1],
                "category": ["error"],
                "statsPeriod": "1d",
                "interval": "1d",
                "field": ["sum(quantity)"],
            }
        )

        assert self.captured_logs[0].organization_id == str(self.organization.id)
```